README.md

```diff
@@ -5,37 +5,40 @@
 [![License](https://img.shields.io/pypi/l/neural-pipeline-search?color=informational)](LICENSE)
 [![Tests](https://github.com/automl/neps/actions/workflows/tests.yaml/badge.svg)](https://github.com/automl/neps/actions)
 
-NePS helps deep learning experts to optimize the hyperparameters and/or architecture of their deep learning pipeline with:
+Welcome to NePS, a powerful and flexible Python library for hyperparameter optimization and neural architecture search. This library is designed to assist researchers and deep learning practitioners in efficiently tuning hyperparameters and exploring neural network architectures to achieve optimal model performance.
 
-- Hyperparameter Optimization (HPO) ([example](neps_examples/basic_usage/hyperparameters.py))
-- Neural Architecture Search (NAS) ([example](neps_examples/basic_usage/architecture.py), [paper](https://openreview.net/forum?id=Ok58hMNXIQ))
-- Joint Architecture and Hyperparameter Search (JAHS) ([example](neps_examples/basic_usage/architecture_and_hyperparameters.py), [paper](https://openreview.net/forum?id=_HLcjaVlqJ))
+## Key Features
 
-For efficiency and convenience NePS allows you to
+In addition to the common features offered by traditional HPO and NAS libraries, NePS stands out with the following key features:
```

Comment:
I shall not merge this yet.

This is a good first iteration to change the landing page. I do like elements of it.
However, the major reason why I shall not merge for now is consistency.

I think we can improve the user experience with every hyperlink directing the user to the same visual experience. That is, we either link everything to another GitHub page or only link to pages in the documentation.

Secondly, if we are putting 5 links right in the beginning, such that someone is more likely to click. The subsequent pages should have more content than what is now. 🙈 So I guess if we want to change the first interface, we can do so only after a little more backend content...

For point 1, either in the main point or as a subpoint, please mention these 2 papers as:
* [PriorBand: Practical Hyperparameter Optimization in the Age of Deep Learning, NeurIPS 2023](https://arxiv.org/abs/2306.12370)
* [πBO: Augmenting Acquisition Functions with User Beliefs for Bayesian Optimization, ICLR 2022](https://arxiv.org/abs/2204.11051)

For point 2, same with:
* [Construction of Hierarchical Neural Architecture Search Spaces based on Context-free Grammars, NeurIPS 2023](https://arxiv.org/abs/2211.01842)

Reply:
Our doc is currently a work in progress. I would personally motivate us linking another GitHub page till we iterate through the docs later again and update this to link to documentation. What do you think ?

Reply:
Papers are now mentioned. Have a look at the current landing page README [here](https://github.com/automl/neps/tree/readme_changes_danrgll)